code/espurna/thinkspeak.ino

```diff
@@ -36,8 +37,25 @@ bool _tspk_flush = false;
 unsigned long _tspk_last_flush = 0;
 unsigned char _tspk_tries = THINGSPEAK_TRIES;
 
+class AsyncThingspeak : public AsyncClient
+{
+  public:
+    URL* address;
```

Comment:
Why use a pointer when it can be a class member?
Minor c++ note: if object has member `_a`, we can have `constructor(T a) : _a(a) {}` syntax.

Reply:
I wasn't use a class member simply because I had not find how to do it. Thanks for your help.